department-of-veterans-affairs/va.gov-cms
# VACMS-7741 claro login updates

## Description

Closes #7741.  Designs in this figma: https://www.figma.com/file/w0z1NIdY7nent0Q6r7hjTD/Login-page?node-id=304%3A3

- Adjusts labels & help text
- Adjust spacing between content & grey bg
- Updates link styles
- Keeps "veteran-centered" on one line
- Updated footer to use svg instead of blurry png, update footer sizing to match figma.
- Added link to VA Design System to footer menu on prod.

## Testing done
Enabled simplesaml login locally to view PIV login button, tested both PIV & form login displays at various screen sizes.

## Screenshots
![Screen Shot 2022-01-27 at 2 02 48 PM](https://user-images.githubusercontent.com/11279744/151454931-68674990-a141-47c9-85a6-4e414605ab93.png)
![Screen Shot 2022-01-27 at 2 05 48 PM](https://user-images.githubusercontent.com/11279744/151454934-62153990-e291-4842-be48-19779b4f8f76.png)
![Screen Shot 2022-01-27 at 2 38 11 PM](https://user-images.githubusercontent.com/11279744/151455120-d0778b29-4c8e-4e58-8c28-edb6f54be15c.png)

## QA steps

As user, visit PR environment and see the new log in page (will switch to use claro theme after tests run... and will work on passing those tests in claro theme next sprint)

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
